rework_pysatl_mpest/core/mixture.py

```diff
@@ -15,7 +15,7 @@
 from numpy.typing import ArrayLike, NDArray
 from scipy.special import logsumexp, softmax
 
-from ..utils.typings import DType
+from rework_pysatl_mpest.typings import DType
```

Comment:
Imports inside of the project is the relative.

```suggestion
from ..typings import DType
```

Reply:
And so on in the other files.